machine/install-machine.md

```diff
@@ -5,12 +5,12 @@ title: Install Docker Machine
 hide_from_sitemap: true
 ---
 
-On macOS and Windows, Machine is installed along with other Docker products when
+On macOS and Windows, Machine is no longer installed along with other Docker products when
```

Comment:
We should probably remove this whole block (feels more like a changelog-item now)

Reply:
Yup, will do a separate PR for that. Thanks.